proto/minder/v1/minder.proto

```diff
@@ -1694,6 +1707,19 @@ message DeleteProjectResponse {
     string project_id = 1;
 }
 
+message CreateEntityReconciliationTaskRequest {
+    // entity is the entity to be reconciled.
+    EntityTypedId entity = 1;
+
+    // context is the context in which the entity reconciliation task is created.
+    Context context = 2;
+}
+
+message CreateEntityReconciliationTaskResponse {
+    // task_id is the id of the task that was created.
+    string task_id = 1;
```

Comment:
Do we provide any use for this `task_id` handle?  I'd rather not provide a handle to internal state if there's nothing to be done with it.

Reply:
Removed `task_id`